test/test_jit.py

```diff
@@ -6708,6 +6708,14 @@ def foo(x):
 
         self.assertEqual(foo(input), input)
 
+    def test_list_builtin(self):
+        @torch.jit.script
+        def fn(x):
+            y = list(x + 1, x + 2, x + 3)
```

Comment:
This is not what the Python semantics are. `list(1, 2, 3)` throws with `list() takes at most 1 argument (3 given)`